packages/moonstone/VideoPlayer/MediaControls.js

```diff
@@ -22,6 +22,15 @@ import {countReactChildren} from './util';
 
 import css from './VideoPlayer.less';
 
+const OuterContainer = SpotlightContainerDecorator({
+	enterTo: 'default-element',
+	defaultElement: [
+		`.${css.leftComponents} .${spotlightDefaultClass}`,
+		`.${css.rightComponents} .${spotlightDefaultClass}`,
+		`.${spotlightDefaultClass}`,
+		`.${css.mediaControls} > .spottable`
```

Comment:
Instead of hard-coding the `spottable` class name, I'd suggest importing it from `spotlight/Spottable`. Better yet, you could update this line to be `.${css.mediaControls} *` since we know each contained control is a spottable component.